stories/Carousel.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
please limit the width of the module, the huge image looks a bit broken

Reply:
The carousel is designed to fill it's container. I limited the storybook example the same way the demo on lightningdesignsystem.com is limited.

Reply:
Salesforce applies `width: 480px` in a wrapper

Reply:
Edit: nvm misread your comment. All good 👍